### PR TITLE
ADFS tweaks

### PR DESCRIPTION
**Pull request by ptbrowne**
## Goals

Polishing login via government ADFS. Each commit should be self explanatory.

- [x] Added variables to Gitlab
- [ ] Removed old keycloak variables (must be done when PR is merged)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,40 +241,12 @@ After the HAR file has been recorded, use [har-to-k6](https://k6.io/docs/test-au
 
 ## Authentication
 
-Authentication by eIAM through a Keycloak instance.
+Authentication is provided by the Swiss federal government's eIAM through ADFS.
 We use Next-auth to integrate our application with Keycloak.
 See https://next-auth.js.org/providers/keycloak for documentation.
```
Comment (adintegra) [inline README.md:246]: I think we could drop references to Keycloak altogether.

Alternatively, we could keep them (including what we had before in the Locally section below) and explain the use of Keycloak as an identity provider. Obviously more effort, but potentially more useful to other users of the project.

**Review by adintegra — APPROVED** [review]
LGTM – thanks! 🎉

File: README.md (hunk 1 above)
Comment (ptbrowne) [reply README.md]: Ah, I had forgotten this last mention. I removed it and pointed to the custom adfs provider. Thanks.

**Comment by adintegra** [comment]
@ptbrowne just saw your comment on the dropping of columns. The columns were added in the [branch](https://github.com/visualize-admin/visualization-tool/tree/extend-user-table) I created (but not yet merged) yesterday. I suspect the migration must have been run from the branch and caused the schema drift.

**Review by bprusinowski — APPROVED** [review]

**Pull request merged**